packages/frontend-2/lib/billing/composables/actions.ts

```diff
@@ -40,9 +45,35 @@ export const useBillingActions = () => {
     })
   }
 
+  const validateCheckoutSession = (workspaceId: string) => {
```

Comment:
In my SSO branch, Fabians told me to always make composables reactive by using refs/maybeRef

`this is why composables that accept a component's data through fn params should always accept a Ref or a MaybeRef so that it can react to changes to that value`